bokehjs/src/lib/models/glyphs/webgl/markers.frag

```diff
@@ -63,6 +63,15 @@ float marker(vec2 P, float size)
 }
 #endif
 
+#ifdef USE_STAR
+// star
+float marker(vec2 P, float size)
+{
+    vec2 q = abs(P);
+    return max(q.y * 0.57735 + q.x - 1.0 * size/2.0, q.y - 0.866 * size/2.0);
+}
+#endif
+
```

Comment:
Unless a webgl shader function for star can be developed, then the changes in this file should be removed. It's better to fall back to non-webgl rendering and draw the correct thing, than to draw the wrong thing in webgl.